content/en/blog/2024/otel-cicd-sig/index.md

```diff
@@ -0,0 +1,143 @@
+
+The need for standardization is relevant for creating the semantic conventions mentioned above, the language for reporting what goes on in the pipeline. Standardization is also needed for the means in which this reporting is propagated through the system, such as upon spawning processes during the pipeline execution. This led us to promote standardization for using environment variables for context and baggage propagation between processes, another important milestone that was recently approved and merged.
+
+## OpenTelemetry: the natural home for CI/CD observability specification
+
+This realization drove us to look for the right way to approach creating a specification. OpenTelemetry emerges as the standard for telemetry generation and collection. The OpenTelemetry specification is tasked with exactly this problem: creating a common uniform and vendor-agnostic specification for telemetry. And housed under the Cloud Native Computing Foundation (CNCF) can ensure it remains open and vendor-neutral. As long standing advocates of OpenTelemetry, it only made sense to extend OpenTelemetry to cover this important DevOps use case. 
+
+We started with an [OpenTelemetry extension proposal (OTEP #223)](https://github.com/open-telemetry/oteps/pull/223) a couple of years ago, proposing our idea to extend OpenTelemetry to cover the CI/CD observability use case. In parallel, we’ve started a Slack channel on the CNCF Slack to gather fellow enthusiasts behind the idea and start brainstorming what that should look like. The Slack channel grew and we quickly discovered that the problem is common across many organizations.
+
+With the feedback from the Technical Oversight Committee and others within the CNCF, we’ve taken the path of asking the mandate to start a dedicated Working Group for the topic under OpenTelemetry’s Semantic Conventions SIG (SIG SemConv in short). With their blessing, we [launched the formal CI/CD Observability SIG](https://github.com/open-telemetry/community/blob/main/projects/ci-cd.md) to formalize our previous Slack group discussions and goals. 
+
+## OpenTelemetry’s CI/CD Observability SIG
+
+Since November of 2023, the SIG has been actively working to develop the standard for semantics around CI/CD observability in collaboration with experts from multiple companies and open source projects. At its inception, we decided to focus on a few key areas for 2024:
+
+* An initial set of common attributes across CI/CD systems.
+* Develop prototype(s) to include both holistic and signal-specific attributes.
+* Carry forward the proposal to add environment variables as context propagators to the OpenTelemetry specification (OTEP #258).
```

Comment:
```suggestion
* Carry forward the proposal to add environment variables as context propagators to the OpenTelemetry specification (OTEP #258).
```

Add a link to the OTEP, see https://github.com/open-telemetry/opentelemetry.io/pull/5718#discussion_r1867942471